src/sidebar/components/sidebar-content.js

```diff
@@ -39,10 +39,9 @@ function SidebarContentController(
    * not the order in which they appear in the document.
    */
   function firstSelectedAnnotation() {
-    if (store.getState().selection.selectedAnnotationMap) {
-      const id = Object.keys(
-        store.getState().selection.selectedAnnotationMap
-      )[0];
+    const selectedAnnotationMap = store.getSelectedAnnotationMap();
+    if (selectedAnnotationMap) {
+      const id = Object.keys(selectedAnnotationMap)[0];
```

Comment:
I note an existing issue that this code _assumes_ all keys in `selectedAnnotationMap` have the value `true`. It seems to me there is a risk that some change in the store module could violate this assumption and break this code without realizing.

Reply:
Yes, the "map"-ness of `selectedAnnotationMap` is problematic. It's treated as an array of truthy things in some places and as an `id` => `selectedOrNot` in other places. I want to re-evaluate overall...